wpilib/wpilib/smartdashboard.py

```diff
@@ -53,41 +64,51 @@ def getTable(cls):
 
     @classmethod
     def putData(cls, *args, **kwargs):
-        """Maps the specified key to the specified value in this table.
+        """
+        Two argument formats are supported: 
+        
+        key, data
+        value
+
+        Maps the specified key (name of the :class:`.Sendable` if not provided) 
+        to the specified value in this table.
```

Comment:
Huh. I only just noticed this. The summary line should come first — if I come here wondering what this method does, the first thing I'd want to see is what it does, not the fact that there are two ways to specify arguments.

Keep the details about the arguments next to the documentation about the arguments.